content/en/metrics/guide/custom_metrics_governance.md

```diff
@@ -0,0 +1,211 @@
+
+   1. From the [Plan & Usage page][2], scroll down to the *Usage Summary* section.
+   1. Click the **Custom Metrics** tab to view your organization's billable usage, usage trends, and top custom metrics.
+   1. Scroll down to the *Top Custom Metrics for \<MONTH YEAR\>*, to see the top metrics that drive up your custom metrics costs.
+
+Additionally, review [Usage Attribution][3] information for a total breakdown of your account's billable usage by tag attributes. From here, you can identify your largest cost drivers by tags such as team, service, or organization. 
+
+**Note**: Usage Attribution is an advanced feature included in the Enterprise plan. For all other plans, contact your account representative or Customer Success to request this feature.
+
+Knowing which metrics are the largest contributor of your account's monthly usage and costs is the recommended starting point for using [Metrics without Limits™][4]. With this knowledge, you can find the source of these metric submissions whether by teams, service, organization, or other tag attribute.
+
+### Team-level visibility and attribution
+
+Individual teams may have limited insights into the costs of the metrics and tags they're submitting. This results in teams being less motivated to control their usage or even limit usage growth. It is crucial for everyone to have visibility into their usage and feel empowered to take ownership in managing those volumes and associated costs. Team-level visibility gives account administrators the ability to hold teams accountable, but more importantly, gives teams the opportunity to understand and reduce their impact on metrics volume.
+
+#### Find out where your largest custom metrics are coming from
+<div class="alert alert-warning">You must have the <a href="https://docs.datadoghq.com/account_management/rbac/?tab=datadogapplication">Datadog Admin Role</a> to access the Plan & Usage page.</div>
+
+{{< img src="metrics/guide/custom_metrics_governance/team_attribution_plan_usage_table.png" alt="Navigate to the Metrics Summary from the Plan & Usage page through the Top Custom Metrics table" style="width:90%;" >}}
+
+To identify which team or service is responsible for any custom metric:
+
+1. From the [Plan & Usage page][2], scroll down to the *Usage Summary* section.
+1. Click the **Custom Metrics** tab to view your organization's billable usage, usage trends, and top custom metrics.
+1. Under the table for *Top Custom Metrics for <MONTH YEAR>*, click  the icon to **See in Metrics Summary** for the top custom metric.
+1. This takes you to the *Metrics Summary* page with the opened metric details sidepanel. 
+1. In the sidepanel, scroll down to the *Tags* section to view associated tags such as teams and service.
+
+You can also view the responsible teams through the [Usage Attribution page][5]. See a breakdown of your account's bill by any arbitrary tag key of your choosing. For example, `team:X` emits a volume of 10K custom metrics. 
```

Comment:
```suggestion
You can also view the responsible teams through the [Usage Attribution page][5]. To see a breakdown of your account's bill, select any tag key. For example, `team:X` emits a volume of 10K custom metrics. 
```

Reply:
generally, `arbitrary` means according to whim instead of reason. We don't mean that here, I'm guessing.